hazelcast/src/main/java/com/hazelcast/monitor/impl/LicenseInfo.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) 2008-2018, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.monitor.impl;
+
+import com.eclipsesource.json.JsonObject;
+
+public class LicenseInfo {
```

Comment:
Why is this class in the monitor,impl package? This package should be used for e..g MapStatisticsImpl. It should not be used as a dumping ground for Member->MC serialization logic.

Reply:
> Why is this class in the monitor,impl package?

Because every other class for Member -> MC serialization logic was already here (such as `WanSyncState` etc). 

Do you think it would be better under for e.g. `com.hazelcast.internal.license`? Any other suggestions?